packages/entities/entities-shared/src/components/entity-base-config-card/ConfigCardItem.vue

```diff
@@ -104,6 +104,7 @@
             </div>
 
             <KTooltip v-else
+              :data-testid="`${props.item.key}-plain-text`"
```

Comment:
Since this broke something, can you add a test to ensure this attribute exists?

Reply:
done and done